packages/js/data/src/crud/selectors.ts

```diff
@@ -113,6 +113,20 @@ export const getItems = createSelector(
 	}
 );
 
+export const getItemsTotalCount = (
+	state: ResourceState,
+	query: ItemQuery | string
```

Comment:
Why can query be an `ItemQuery` or a `string` here?

Reply:
See response in the previous comment: https://github.com/woocommerce/woocommerce/pull/34613#discussion_r966385117 which explains it.

Reply:
I end up removing the `| string` as I added the `defaultArgs` support.